core/jvm/src/main/scala/scalaz/zio/App.scala

```diff
@@ -27,12 +25,7 @@ import scala.concurrent.duration.Duration
  */
 trait App extends RTS {
 
-  sealed trait ExitStatus
-  object ExitStatus {
-    case class ExitNow(code: Int)                         extends ExitStatus
-    case class ExitWhenDone(code: Int, timeout: Duration) extends ExitStatus
-    case object DoNotExit                                 extends ExitStatus
-  }
+  case class ExitStatus(code: Int)
```

Comment:
Let's separate out this change into a separate PR, since I think there's some thought that